### ISSUE TITLE
need feature to skip hooks during rebase

### ISSUE THREAD

**Issue by jt3k** [ForkIssues/Tracker#777]
can't finish rebase
![](https://i.imgur.com/cje9Y2P.png)

<details>
<summary></summary>

777 get :)

</details>

**Comment by DanPristupov** [comment]
<img width="458" alt="Screenshot 2019-10-18 at 13 54 28" src="https://user-images.githubusercontent.com/618115/67092155-cef0c580-f1ae-11e9-8102-0c8578ddbd58.png">

**Comment by DanPristupov** [comment]
Sorry, didn't notice it was about rebase first.

Regarding the issue. Do you mean the `pre-rebase` hook?

**Comment by DanPristupov** [comment]
Anyway you can create a custom branch command for that.

**Comment by jt3k** [comment]
I did not fully understand precommit or another hook.
I clicked on the early commit with the right mouse button and selected "rebase interactive to here ..." renamed 2 commits, and clicked Rebase. An error warning has appeared. after that, the rebase was not completed.


in other cases, when I commit and git-hooks errors appear, I have a "skip hooks and commit" button here.

I miss this button when rebase :D
